### ISSUE TITLE
wizard button missing when importing from sqlite

### ISSUE THREAD

**Issue by azazellochg** [I2PC/scipion#1940]
**Describe the bug**
When importing particles from scipion / sqlite file there is no wizard button anymore....
I didn't check other protocols

**Screenshots**
![screenshot_20190227_135428](https://user-images.githubusercontent.com/6952870/53495189-51e5d400-3a97-11e9-99dd-e20760f0ada0.png)

**Comment by pconesa** [comment]
I got this, not only that one but 3 more!

The reason is, they are all in wizard.py and not wizards.py, so the Domain.getWizards is skiping those classes.

So, how to fix this:

A.- I've moved wizard.py to wizards.py and wizard.py has 1 single import line: from wizards import * (otherwise all plugin's import from em.wizard will break.

B.- Move the 4 missing ones to wizards.py (I'm not sure if any plugin is inheriting from any of those ) and:
    B.1: Import them in wizard.py (wait, I think this will create an import loop)
    B.2: Do nothing else, if any plugin is inheriting from any of those we'll find out.

The wizards are:
ImportOriginVolumeWizard
ImportCoordinatesBoxSizeWizard
GetStructureChainsWizard (base class for GetStructureChainsWizardAtomStruct, and GetStructureChainsWizardChimera)
ImportAcquisitionWizard
